src/utils/__tests__/getFlowType-test.ts

```diff
@@ -1269,4 +1269,96 @@ describe('getFlowType', () => {
       raw: '{ subAction: SubAction }',
     });
   });
+
+  it('handles ObjectTypeSpreadProperty', () => {
+    const typePath = statement(`
+      var x: {| apple: string, banana: string, ...OtherFruits |} = 2;
+      type OtherFruits = { orange: string }
+    `)
+      .get('declarations', 0)
+      .get('id')
+      .get('typeAnnotation')
+      .get('typeAnnotation');
+
+    expect(getFlowType(typePath, null, noopImporter)).toEqual({
+      name: 'signature',
+      type: 'object',
+      signature: {
+        properties: [
+          {
+            key: 'apple',
+            value: {
+              name: 'string',
+              required: true,
+            },
+          },
+          {
+            key: 'banana',
+            value: {
+              name: 'string',
+              required: true,
+            },
+          },
+          {
+            key: 'orange',
+            value: {
+              name: 'string',
+              required: true,
+            },
+          },
+        ],
+      },
+      raw: '{| apple: string, banana: string, ...OtherFruits |}',
+    });
+  });
+
+  it('handles ObjectTypeSpreadProperty from imported types', () => {
+    const typePath = statement(`
+      var x: {| apple: string, banana: string, ...MyType |} = 2;
+      import type { MyType } from 'MyType';
+    `)
+      .get('declarations', 0)
+      .get('id')
+      .get('typeAnnotation')
+      .get('typeAnnotation');
+
+    expect(getFlowType(typePath, null, mockImporter)).toEqual({
```

Comment:
For the 2 new test can you please use `toMatchSnapshot()`.
I know the other tests don't use it, but using it keeps the tests smaller and readable.

Reply:
Great suggestion!

Reply:
What was the reasoning to use `toMatchInlineSnapshot` in favor of `toMatchSnapshot`? Did the later one not work?

Reply:
Oops sorry I mistakenly assumed that we wanted to put the snapshot together with the cases. Just updated to use `toMatchSnapshot` and found there're other test cases using `toMatchSnapshot` already 😄